client/v2/autocli/msg.go

```diff
@@ -130,13 +121,32 @@ func (b *Builder) BuildMsgMethodCommand(descriptor protoreflect.MethodDescriptor
 			return err
 		}
 
-		return b.outOrStdoutFormat(cmd, bz)
+		// enable sign mode textual
+		b.TxConfigOpts.EnabledSignModes = append(b.TxConfigOpts.EnabledSignModes, signing.SignMode_SIGN_MODE_TEXTUAL)
+		b.TxConfigOpts.TextualCoinMetadataQueryFn = authtxconfig.NewGRPCCoinMetadataQueryFn(clientCtx)
+
+		txConfigWithTextual, err := authtx.NewTxConfigWithOptions(
+			codec.NewProtoCodec(clientCtx.InterfaceRegistry),
+			b.TxConfigOpts,
+		)
+		if err != nil {
+			return err
+		}
+		clientCtx = clientCtx.WithTxConfig(txConfigWithTextual)
+
+		msg := dynamicpb.NewMessage(input.Descriptor())
+		proto.Merge(msg, input.Interface())
+
+		return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
```

Comment:
Getting a small issue here. It looks like we cannot properly json marshal a dynamic message.
Opening the PR already for feedback.